code/modules/shuttles/shuttle_supply.dm

```diff
@@ -77,7 +84,7 @@
 				lower_railings()
 				return
 		else //at centcom
-			GLOB.supply_controller.buy()
+			GLOB.supply_controller_upp.buy() //FIX THIS HOLY FUCK FIX!
```

Comment:
?

Reply:
ah jeh that is there as we need elevators reworked to new shuttle code